pages/index.tsx

```diff
@@ -268,7 +269,7 @@ export const HomePage: NextPage = () => {
             </Box>
           </Flex>
         </Flex>
-        <Grid templateColumns="repeat(3, 1fr)" gap="32px" py="32px">
+        <SimpleGrid minChildWidth="600px" gap="32px" py="32px">
```

Comment:
Why the switch to `SimpleGrid`? This has resources display in a single column on every screen size, and 600px is wider than mobile screens.

Not sure if you have access to access to [the Figma designs](https://www.figma.com/file/weR9JkmoDTVtQMuVeBViXa/Designs?type=design&node-id=413-53512&t=ggtr4ZgwGATLWxk8-0) but just sent you an invite!

Reply:
I just read the section on "auto-responsive grid" in the SimpleGrid doc and tried it out (when that seemed to work, I then just tried the same minWidth trick on every stack/box that looked like it should be longer and at least in the dev tools that seemed to fix it). Will switch back to a normal grid and using the break point stuff to set the number of columns.